pkg/kubecost/allocation.go

```diff
@@ -65,7 +65,7 @@ type Allocation struct {
 	GPUCostAdjustment      float64               `json:"gpuCostAdjustment"`
 	NetworkCost            float64               `json:"networkCost"`
 	LoadBalancerCost       float64               `json:"loadBalancerCost"`
-	PVBreakdown            map[string]PVUsage    `json:"pvBreakDown"`
+	PVs                    PV
```

Comment:
We'll probably want this to have `json:"-"` to explicitly indicate not to JSON encode/decode, and for consistency.